src/flex-example-app-android.xml

```diff
@@ -72,6 +76,14 @@
                     <activity android:name="com.vungle.publisher.FullScreenAdActivity" android:configChanges="keyboardHidden|orientation|screenSize" android:theme="@android:style/Theme.NoTitleBar.Fullscreen"    />
                     <service android:name="com.vungle.publisher.VungleService" android:exported="false"/>
 
+
+<activity android:name="com.inmobi.rendering.InMobiAdActivity" android:configChanges="keyboardHidden|orientation|keyboard|smallestScreenSize|screenSize" android:theme="@android:style/Theme.Translucent.NoTitleBar" android:hardwareAccelerated="true" /><receiver android:name="com.inmobi.commons.core.utilities.uid.ImIdShareBroadCastReceiver" android:enabled="true" android:exported="true" >
```

Comment:
also the same question here about the receiver - it's not in their docs, is it required?